packages/smooth_app/lib/cards/product_cards/smooth_product_card_found.dart

```diff
@@ -89,58 +79,54 @@ class SmoothProductCardFound extends StatelessWidget {
-                    mainAxisAlignment: MainAxisAlignment.spaceBetween,
-                    crossAxisAlignment: CrossAxisAlignment.start,
-                    children: <Widget>[
-                      SizedBox(
-                        width: screenSize.width * 0.65,
-                        child: Column(
+                  child: Container(
+                    height: screenSize.width * 0.2,
+                    child: Column(
+                      mainAxisAlignment: MainAxisAlignment.spaceBetween,
+                      crossAxisAlignment: CrossAxisAlignment.start,
+                      children: <Widget>[
+                        Text(
+                          product.productName ?? '???',
+                          overflow: TextOverflow.ellipsis,
+                          style: const TextStyle(
+                            fontSize: 18.0,
+                            fontWeight: FontWeight.bold,
+                          ),
+                        ),
+                        Text(
+                          product.brands ?? '???',
+                          overflow: TextOverflow.ellipsis,
+                        ),
+                        Row(
                           children: <Widget>[
-                            Row(
-                              mainAxisAlignment: MainAxisAlignment.spaceBetween,
-                              children: <Widget>[
-                                Flexible(
-                                  child: Text(
-                                    productTitle,
-                                    maxLines: 2,
-                                    overflow: TextOverflow.fade,
-                                  ),
-                                ),
-                                if (handle != null) handle!,
-                              ],
-                            ),
-                            SizedBox(
-                              width: screenSize.width * 0.65,
-                              child: Wrap(
-                                direction: Axis.horizontal,
-                                children: scores,
-                                spacing: 2.0,
-                                runSpacing: 2.0,
-                              ),
+                            Icon(
+                              Icons.circle,
+                              size: 15,
+                              color:
+                                  getProductCompatibilityHeaderBackgroundColor(
+                                      compatibility),
                             ),
+                            const SizedBox(width: 5),
+                            Text(getProductCompatibilityHeaderTextWidget(
+                                compatibility)),
                           ],
                         ),
-                      ),
-                    ],
+                      ],
+                    ),
                   ),
                 ),
+                const SizedBox(width: 5),
```

Comment:
padding might be a better fit

Reply:
Done.

Why is padding better?

Reply:
just feels like it was created for this use case.

Reply:
Check